### PR TITLE
fix: scroll enable on smaller screen

### PR DESCRIPTION
**Pull request by leangseu-edx**
TLDR; make sure the response display scrollable on small screen

[AU-513](https://openedx.atlassian.net/browse/AU-513)

- Fix small screen scrolling

https://user-images.githubusercontent.com/83240113/151281117-56f7a8d4-e708-4ff0-80ae-0564b0eb50d1.mov

- Fix error banner position on response display
Screen Shot 2022-01-27 at 1 22 58 PM
Screen Shot 2022-01-27 at 1 22 52 PM

- make rubric and review error width align
Screen Shot 2022-01-27 at 1 48 10 PM

- make rubric sticky
Screen Shot 2022-01-27 at 1 52 02 PM

### Additional fix from meeting
- fix miss align rubic and submission file on the top
- fix padding size on small screen
- add optional on feedback comment for clarity

### Records after UI fixes

https://user-images.githubusercontent.com/83240113/151439104-26db0d31-44c8-4b24-893c-bcfa364fa375.mov

https://user-images.githubusercontent.com/83240113/151439117-7d1a4673-3178-4352-ad56-2618f6876279.mov

### CODE REVIEW

**Comment by mattcarter** [comment]
@leangseu-edx can you include screenshots please

**Comment by leangseu-edx** [comment]
> @leangseu-edx can you include screenshots please

I made a video, not sure it would be a clear presentation.

File: src/containers/CriterionContainer/CriterionFeedback.test.jsx [hunk 1]
```diff
@@ -67,11 +67,13 @@ describe('Criterion Feedback', () => {
       expect(el.instance().render()).toMatchSnapshot();
     });
 
-    test('is configure to disabled', () => {
-      el.setProps({
-        config: feedbackRequirement.disabled,
+    Object.values(feedbackRequirement).forEach(requirement => {
+      test(`feedback is configure to ${requirement}`, () => {
```
Comment (muselesscreator) [inline src/containers/CriterionContainer/CriterionFeedback.test.jsx]:
```suggestion
      test(`feedback is configured to ${requirement}`, () => {
```

File: src/containers/CriterionContainer/CriterionFeedback.jsx [hunk 2]
```diff
@@ -25,6 +25,15 @@ export class CriterionFeedback extends React.Component {
     });
   }
 
+  get commentMessage() {
```
Comment (muselesscreator) [inline src/containers/CriterionContainer/CriterionFeedback.jsx:28]: this get logic needs a test.

**Review by muselesscreator — CHANGES_REQUESTED** [review]
looks great! 1 minor spelling nit and 1 missing new test.

File: src/containers/CriterionContainer/CriterionFeedback.test.jsx [hunk 3]
```diff
@@ -121,6 +126,18 @@ describe('Criterion Feedback', () => {
         expect(props.setValue).toBeCalledTimes(1);
       });
     });
+
+    test('getter commentMessage', () => {
+      el.setProps({ config: feedbackRequirement.optional });
```
Comment (muselesscreator) [inline src/containers/CriterionContainer/CriterionFeedback.test.jsx]: this should be a describe, and it should test the conditional inclusion of the 'optional' message as well as `comments` vs `addComments`

**Review by muselesscreator — APPROVED** [review]
LGTM
